#! /bin/sh /usr/share/dpatch/dpatch-run
## 50_debianize.dpatch by jre <jre-phoenix@users.sourceforge.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: - Change moblock-control.defaults and moblock.conf to default to full
## DP:   Debian compatibility.

@DPATCH@
diff -urNad moblock-control~/moblock-control.defaults moblock-control/moblock-control.defaults
--- moblock-control~/moblock-control.defaults	2009-01-05 17:51:22.000000000 +0100
+++ moblock-control/moblock-control.defaults	2009-01-09 16:43:08.000000000 +0100
@@ -70,5 +70,5 @@
 PIDFILE="/var/run/$NAME.pid"
 LSB="/lib/lsb/init-functions"
 TESTHOST="iblocklist.com"
-LSB_MODE="1"
+LSB_MODE="0"
 CRON_EMAILTO="root"
diff -urNad moblock-control~/moblock.conf moblock-control/moblock.conf
--- moblock-control~/moblock.conf	2009-01-09 16:13:57.000000000 +0100
+++ moblock-control/moblock.conf	2009-01-09 16:43:39.000000000 +0100
@@ -330,8 +330,8 @@
 # you can download one based on the Debian version from
 # moblock-deb.sourceforge.net.
-# 0 - Debian compatible system (uses start-stop-daemon instead of start_daemon)
-# 1 - LSB 3.1 compatible system (default)
+# 0 - Debian compatible system (uses start-stop-daemon instead of start_daemon) (default)
+# 1 - LSB 3.1 compatible system
 # 2 - system with broken LSB 3.1 implementation, which doesn't understand the
 #     specification of the pidfile. Use this for Fedora Core 9 (and other redhat
 #     variants).
-LSB_MODE="1"
+LSB_MODE="0"
